src/screens/ImageScreen.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Text, StyleSheet, View } from "react-native";
+import ImageDetail from "../components/ImageDetail";
+
+const ImageScreen = () => {
+  return (
+    <View style={styles.display}>
+      <ImageDetail
+        title="Forest"
+        imageSource={require("../../assets/forest.jpg")}
```

Comment:
To make the code cleaner. please import the images along with other imports on top of the page:

```typescript
// Import images at the top
import forestImage from "../../assets/forest.jpg";
import beachImage from "../../assets/beach.jpg";
import mountainImage from "../../assets/mountain.jpg";

const ImageScreen = () => {
  return (
    <View style={styles.display}>
      <ImageDetail
        title="Forest"
        imageSource={forestImage}
        score={"9"}
      />
      <ImageDetail
        title="Beach"
        imageSource={beachImage}
        score={"6"}
      />
      <ImageDetail
        title="Mountain"
        imageSource={mountainImage}
        score={"3"}
      />
    </View>
  );
};

```